File: pre_for_assess/instantiations/person.rb
class Person
  # def initialize(age, name)
  #   @age = age;
  #   @name = name;
  # end

  def initialize(age, name)
    @age = age;
    @name = name;
    class << self
      def act
        puts "hello"
      end
    end
  end

  def act
    behavior = @name + " is " + @age.to_s + " years old and he is walking.";
    puts behavior;
  end
end

p = Person.new(18, 'Bob')
p.act

File: pre_for_assess/instantiations/person.py
class Person:
    def __init__(self, age, name):
        self.age = age
        self.name = name

    def act(self):
        behavior = self.name + " is " + str(self.age) + " years old and he is walking.";
        print(behavior)

p = Person(18, 'Bob')
p.act()
